compiler/gradient_with_order.cc

```diff
@@ -190,6 +190,9 @@ void AddGradientNodesForTrainingWithOrders(Graph* graph, const std::vector<Order
                 // Copy gradients of outputs from the original
```

Comment:
Could you update the comment to mention inputs are also needed to be copied?
```
                // Copy gradients of inputs/outputs from the original
                // computation node to the last forward computation.
                // Copying inputs is necessary to accumulate gradients.
```

Reply:
updated.